sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileClient.java

```diff
@@ -226,7 +226,7 @@ public PathInfo flush(long position) {
      */
     public PathInfo flush(long position, boolean overwrite) {
         DataLakeRequestConditions requestConditions = new DataLakeRequestConditions();
-        if (!overwrite) {
```

Comment:
Why was this changed?

Reply:
Man, you were soooo close to preventing this bug from being released